shell/platform/darwin/ios/framework/Source/FlutterEngine.mm

```diff
@@ -576,7 +587,7 @@ - (instancetype)initWithPlugin:(NSString*)pluginKey flutterEngine:(FlutterEngine
   self = [super init];
   NSAssert(self, @"Super init cannot be nil");
   _pluginKey = [pluginKey retain];
-  _flutterEngine = [flutterEngine retain];
+  _flutterEngine = flutterEngine;//[flutterEngine retain];
```

Comment:
Here and elsewhere: please remove the commented out code.

Reply:
okay

Reply:
done